src/vmm/src/builder.rs

```diff
@@ -1258,6 +1263,23 @@ pub mod tests {
             .is_some());
     }
 
+    pub(crate) fn insert_entropy_device(
+        vmm: &mut Vmm,
+        cmdline: &mut Cmdline,
+        event_manager: &mut EventManager,
+        entropy_config: EntropyDeviceConfig,
+    ) {
+        let mut builder = EntropyDeviceBuilder::new();
+        let entropy = builder.build(entropy_config).unwrap();
+
+        assert!(attach_entropy_device(vmm, cmdline, event_manager, &entropy).is_ok());
+
+        assert!(vmm
+            .mmio_device_manager
+            .get_device(DeviceType::Virtio(TYPE_RNG), ENTROPY_DEV_ID)
+            .is_some());
+    }
```

Comment:
Nit it is usually better to match the specific value e.g. assert_eq!(..., Some(...)).

Reply:
Not sure if it's doable but it might be just me not understanding really well trait objects. The compiler complains to me that `dyn devices::BusDevice + 'static` does not implement `Debug`